src/homepageExperience/components/steps/arduino/InitializeClient.tsx

```diff
@@ -95,22 +96,110 @@ export const InitializeClient: FC<OwnProps> = ({
+  #include <WiFiMulti.h>
+  WiFiMulti wifiMulti;
+  #define DEVICE "ESP32"
+  #elif defined(ESP8266)
+  #include <ESP8266WiFiMulti.h>
+  ESP8266WiFiMulti wifiMulti;
+  #define DEVICE "ESP8266"
+  #endif
+  
+  #include <InfluxDbClient.h>
+  #include <InfluxDbCloud.h>
+  
+  // WiFi AP SSID
+  #define WIFI_SSID "YOUR_WIFI_SSID"
+  // WiFi password
+  #define WIFI_PASSWORD "YOUR_WIFI_PASSWORD"
+  
+  #define INFLUXDB_URL "${url}"
+  #define INFLUXDB_TOKEN "${token}"
+  #define INFLUXDB_ORG "${org.id}"
+  #define INFLUXDB_BUCKET "${
+    bucket.name === '<BUCKET>' ? 'YOUR_BUCKET' : bucket.name
+  }"
+  
+  // Time zone info
+  #define TZ_INFO "UTC−07:00"
+  
+  // Declare InfluxDB client instance with preconfigured InfluxCloud certificate
+  InfluxDBClient client(INFLUXDB_URL, INFLUXDB_ORG, INFLUXDB_BUCKET, INFLUXDB_TOKEN, InfluxDbCloud2CACert);
+  
+  // Declare Data point
+  Point sensor("wifi_status");
+  
+  void setup() {
+    Serial.begin(115200);
+  
+    // Setup wifi
+    WiFi.mode(WIFI_STA);
+    wifiMulti.addAP(WIFI_SSID, WIFI_PASSWORD);
+  
+    Serial.print("Connecting to wifi");
+    while (wifiMulti.run() != WL_CONNECTED) {
+      Serial.print(".");
+      delay(100);
+    }
+    Serial.println();
+  
+    // Accurate time is necessary for certificate validation and writing in batches
+    // We use the NTP servers in your area as provided by: https://www.pool.ntp.org/zone/
+    // Syncing progress and the time will be printed to Serial.
+    timeSync(TZ_INFO, "pool.ntp.org", "time.nis.gov");
+  
+  
+    // Check server connection
+    if (client.validateConnection()) {
+      Serial.print("Connected to InfluxDB: ");
+      Serial.println(client.getServerUrl());
+    } else {
+      Serial.print("InfluxDB connection failed: ");
+      Serial.println(client.getLastErrorMessage());
+    }
+  }`
+
   // Events log handling
   const logCopyCodeSnippet = () => {
-    event(`firstMile.arduinoWizard.buckets.code.copied`)
+    event(`firstMile.arduinoWizard.initializeClient.code.copied`)
   }
 
-  return <h1>Initialize Client</h1>
+  return (
+    <>
+      <h1>Initialize Client</h1>
+      <p>Select or create a bucket to write data to.</p>
+      <Panel backgroundColor={InfluxColors.Grey15}>
+        <Panel.Body size={ComponentSize.ExtraSmall}>
+          <Grid>
+            <Grid.Row>
+              <Grid.Column widthSM={Columns.Twelve}>
+                <WriteDataHelperBuckets useSimplifiedBucketForm={true} />
+              </Grid.Column>
+            </Grid.Row>
+          </Grid>
+        </Panel.Body>
+      </Panel>
+      <p className="small-margins">
+        Paste the following snippet into a blank Arduino sketch file.
+      </p>
+      <CodeSnippet
+        text={codeSnippet}
+        onCopy={logCopyCodeSnippet}
+        language="arduino"
+      />
+      <p style={{fontSize: '14px', marginTop: '8px', marginBottom: '48px'}}>
+        Note: you will need to set the WIFI_SSID and WIFI_PASSWORD variables to
```

Comment:
I think we'll want to wrap `WIF_SSID` and `WIFI_PASSWORD` in `<code className="homepage-wizard--code-highlight">` blocks, like [here](https://github.com/influxdata/ui/blob/master/src/homepageExperience/components/steps/python/InstallDependencies.tsx#L17)

Reply:
Good point, fixed!